src/EditorFeatures/Core/Shared/Tagging/TagSources/ProducerPopulatedTagSource.cs

```diff
@@ -675,7 +686,7 @@ public override ITagSpanIntervalTree<TTag> GetTagIntervalTreeForBuffer(ITextBuff
 
                     // We shall synchronously compute tags.
                     var producedTags = ConvertToTagTree(
```

Comment:
ConvertToTagTree probably should get the cancellation token now? I'm actually surprised it doesn't already take one...